src/lambdaisland/deep_diff2/strip.cljc

```diff
@@ -0,0 +1,43 @@
+(ns lambdaisland.deep-diff2.strip
```

Comment:
I think this should be called `lambdaisland.deep-diff2.manipulate`, or something else that's broader than "strip."